tpm2/tpm2_test.go

```diff
@@ -727,7 +727,7 @@ func TestSign(t *testing.T) {
 					Hash: AlgSHA256,
 				},
 				CurveID: CurveNISTP256,
-				Point:   ECPoint{X: big.NewInt(0), Y: big.NewInt(0)},
+				Point:   ECPoint{XRaw: big.NewInt(0).Bytes(), YRaw: big.NewInt(0).Bytes()},
```

Comment:
Just delete this line (the default value of `XRaw=nil, YRaw=nil` is fine here)

Reply:
Done